RocketChat/Rocket.Chat.ReactNative
# fix: suggestions not shown in second line of multi-line message

## Proposed changes

When writing a multi-line message the emoji, users, and channel suggestions are shown in the first line, but not in the second one, and more.

## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4973

## How to test or reproduce

1. Start typing some emoji, e.g. `:gri`. Observe suggestions for emojis being shown.
2. Select the `:grinning:` emoji and type a new line.
3. Start typing another emoji, e.g. again `:gri`. Observe that no suggestions for emojis are being shown.

## Screenshots

## Types of changes




## Checklist



## Further comments

[TC-544]

[TC-544]: https://rocketchat.atlassian.net/browse/TC-544?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
